### PR TITLE
[improve] Design of architecture

### PR DESCRIPTION
**Pull request by Kodalien**

### CODE REVIEW

**Comment by Kodalien** [comment]
@zhongjiajie  hi,brothher updates

**Comment by zhongjiajie** [comment]
Hi @Kodalien we are going to relase 2.0.3 doc in this weekly, could this PR wait until PR https://github.com/apache/dolphinscheduler-website/pull/646 merged and change version 203 doc?

**Comment by zhongjiajie** [comment]
BTW, we should change dev docs too.

**Comment by Kodalien** [comment]
> 嗨@Kodalien，我们将在这周发布 2.0.3 文档，这个 PR 可以等到 PR #646合并并更改版本 203 文档吗？
okay

**Comment by zhongjiajie** [comment]
@Kodalien Thanks

**Comment by Kodalien** [comment]
> 顺便说一句，我们也应该更改开发文档。
all has changed , I will continue

File: docs/zh-cn/2.0.2/user_doc/architecture/load-balance.md [hunk 1]
```diff
@@ -1,58 +1,52 @@
 ### 负载均衡
-负载均衡即通过路由算法（通常是集群环境），合理的分摊服务器压力，达到服务器性能的最大优化。
 
-### DolphinScheduler-Worker 负载均衡算法
+### Master至Worker 负载均衡算法
```
Comment (QuakeWang) [inline docs/zh-cn/2.0.2/user_doc/architecture/load-balance.md:3]: As the demanded by this [page](https://dolphinscheduler.apache.org/en-us/community/development/document.html). It is recommended that spaces be required between Chinese characters and English and numbers, and that no spaces be required between Chinese punctuation marks and English and numbers, to enhance the aesthetics and readability of the mixed Chinese and English layout. The same format applies to the following.

Comment (Kodalien) [reply docs/zh-cn/2.0.2/user_doc/architecture/load-balance.md:3]: I feel your suggestion very much!

**Comment by zhongjiajie** [comment]
Hi @Kodalien, we just release doc version 2.0.3, do you mind change version 2.0.3 instead of version 2.0.2?

**Comment by Kodalien** [comment]
I have made all modifications on 2.02 !
but,I need to see what is the difference between&nbsp;










kodalien
***@***.***


---Original---
From: "Jiajie ***@***.***&gt;
Date: Tue, Jan 25, 2022 17:12 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [apache/dolphinscheduler-website] updates (PR #649)




 
Hi @Kodalien, we just release doc version 2.0.3, do you mind change version 2.0.3 instead of version 2.0.2?
 
—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Kodalien** [comment]
> 嗨@Kodalien，我们刚刚发布了 2.0.3 版的文档，您介意更改 2.0.3 版而不是 2.0.2 版吗？

I will finish

File: docs/zh-cn/2.0.2/user_doc/guide/task/map-reduce.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# MapReduce(MR)节点
+# MapReduce节点
```
Comment (zhongjiajie) [inline docs/zh-cn/2.0.2/user_doc/guide/task/map-reduce.md:1]: You should also change version 2.0.3 and dev document

**Review by zhongjiajie — CHANGES_REQUESTED** [review]
Hi, I notice you just change version 2.0.2 doc. Since we have already release version 2.0.3, please also change 2.0.3 and dev version document. For 2.0.2, you could still keep 2.0.2 changed or revert it

File: docs/zh-cn/2.0.2/user_doc/guide/task/map-reduce.md (hunk 2 above)
Comment (Kodalien) [reply docs/zh-cn/2.0.2/user_doc/guide/task/map-reduce.md:1]: ok! I will end at all the needs at the end of February. @zhongjiajie

File: docs/zh-cn/2.0.2/user_doc/architecture/design.md [hunk 3]
```diff
@@ -1,116 +1,67 @@
-## 系统架构设计
-本章节介绍Apache DolphinScheduler调度系统架构
+架构概念
+=============================
```
Comment (zhongjiajie) [inline docs/zh-cn/2.0.2/user_doc/architecture/design.md:2]: Same length would be better
```suggestion
架构概念
======
```

File: docs/zh-cn/2.0.2/user_doc/architecture/design.md [hunk 4]
```diff
@@ -1,116 +1,67 @@
...
   </p>
 </p>
 
-#### 1.2 启动流程活动图
-<p align="center">
-  <img src="/img/master-process-2.0-zh_cn.png" alt="Start process activity diagram"  width="70%" />
-  <p align="center">
-        <em>启动流程活动图</em>
-  </p>
-</p>
```
Comment (zhongjiajie) [inline docs/zh-cn/2.0.2/user_doc/architecture/design.md:21]: If you delete img assert, you should delete the img file if without use in others place

File: docs/zh-cn/2.0.2/user_doc/architecture/design.md [hunk 5]
```diff
@@ -1,116 +1,67 @@
...
-
-- 一旦Master出现了问题，则群龙无首，整个集群就会崩溃。为了解决这个问题，大多数Master/Slave架构模式都采用了主备Master的设计方案，可以是热备或者冷备，也可以是自动切换或手动切换，而且越来越多的新系统都开始具备自动选举切换Master的能力,以提升系统的可用性。
-- 另外一个问题是如果Scheduler在Master上，虽然可以支持一个DAG中不同的任务运行在不同的机器上，但是会产生Master的过负载。如果Scheduler在Slave上，则一个DAG中所有的任务都只能在某一台机器上进行作业提交，则并行任务比较多的时候，Slave的压力可能会比较大。
-
-
-
-###### 去中心化
- <p align="center">
-   <img src="https://analysys.github.io/easyscheduler_docs_cn/images/decentralization.png" alt="去中心化"  width="50%" />
- </p>
```
Comment (zhongjiajie) [inline docs/zh-cn/2.0.2/user_doc/architecture/design.md:99]: Same as I comment above when you remove img content

**Review by zhongjiajie — COMMENTED** [review]
Add some addition suggestion here.

**Pull request closed without merging**